---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    entando.org/entando-resource-namespace: "my-namespace"
    entando.org/entando-resource-kind: "EntandoApp"
    EntandoApp: "my-app"
  name: "my-app-85632404"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "entando.org/v1"
    kind: "EntandoApp"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "9174d27d-7b90-4e71-bf27-8db49cd4bb2f"
action: "FAILED"
count: 1
firstTimestamp: "2021-07-10T12:39:51Z"
involvedObject:
  apiVersion: "entando.org/v1"
  kind: "EntandoApp"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "7427916"
  uid: "9174d27d-7b90-4e71-bf27-8db49cd4bb2f"
lastTimestamp: "2021-07-10T12:39:51Z"
message: "The deployment of EntandoApp my-namespace/my-app failed due to Fake Controller\
  \ Exception. Fix the root cause and then trigger a redeployment by adding the annotation\
  \ 'entando.org/processing-instruction: force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "entando-k8s-entando-k8s-app-con-9pvxm-44-from-build-pack--82zqp"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-app-controller"
type: "Error"
